src/components/AccountSwitcher.tsx

```diff
@@ -0,0 +1,146 @@
+import {Str} from 'expensify-common';
+import React, {useRef, useState} from 'react';
+import {View} from 'react-native';
+import {useOnyx} from 'react-native-onyx';
+import useCurrentUserPersonalDetails from '@hooks/useCurrentUserPersonalDetails';
+import useLocalize from '@hooks/useLocalize';
+import usePermissions from '@hooks/usePermissions';
+import useResponsiveLayout from '@hooks/useResponsiveLayout';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import {connect} from '@libs/actions/Delegate';
+import {getPersonalDetailByEmail} from '@libs/PersonalDetailsUtils';
+import variables from '@styles/variables';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import Avatar from './Avatar';
+import Icon from './Icon';
+import * as Expensicons from './Icon/Expensicons';
+import MenuItem from './MenuItem';
+import type {MenuItemProps} from './MenuItem';
+import Popover from './Popover';
+import {PressableWithFeedback} from './Pressable';
+import Text from './Text';
+
+function AccountSwitcher() {
```

Comment:
We can troubleshoot this more once the commands are on staging but right now - this does not seem to load when the user is copiloted into another account

![image](https://github.com/user-attachments/assets/25009e63-c286-4448-b7cf-c1f05aae855b)

I'm not sure if there's an Onyx update we need to send from the backend that I'm not sending yet, or if it's a front end thing. No action yet, but calling it out.

Reply:
We'll have to experiment with which keys to preserve when clearing onyx - KEYS_TO_PRESERVE

Reply:
This has been resolved.